plugins/info/info.h

```diff
@@ -6,34 +6,69 @@ namespace dronecore {
 
 class InfoImpl;
 
+/**
+ * The Info class provides basic info about the hardware and/or software of a device.
+ */
 class Info
 {
 public:
+    /**
+     * Constructor for Info called internally.
+     */
     explicit Info(InfoImpl *impl);
+
+    /**
+     * Destructor for Info called internally.
+     */
     ~Info();
 
-    // 16 chars + terminating 0
+    /**
+     * Length of git hash strings.
```

Comment:
Is this enough information? I don't know where/how you would use this.

Reply:
It's really just a define that is used in the struct below.